gnomad/resources/grch38/gnomad.py

```diff
@@ -428,7 +428,7 @@ def gnomad_gks(
     by_ancestry_group: bool = False,
     by_sex: bool = False,
     vrs_only: bool = False,
-    custom_ht_path: str = None,
+    ht: Union[str, hl.Table] = None,
```

Comment:
I added the ability to pass in a hail table because it made development much easier. I can pass in a much smaller table with only certain variants and run the same code on it. There are some prereqs for that table, like it having the `.info.vrs` struct that is part of the prerelease table. Might be good to specify that in the docstring or somehow check for that when this function is run. It will error out if that struct doesn't exist in the table.

Reply:
maybe to make this more clear there can be two optional params: 'custom_ht' and 'custom_ht_path'?

Reply:
I'd be happy to make there be two optional params and throw an exception if both are provided if that's preferred. I could also see removing the ability to pass a path entirely, and make the user call `hl.read_table` on it themself, which would limit to these cases:
1. no custom_ht is provided, load the default one
2. user provides a custom_ht that they have instantiated and done whatever annotations/filtering to beforehand

Reply:
ok, lets do the second option (only the option of a custom ht)

Reply:
@klaricch I just implemented this in #559 here: https://github.com/broadinstitute/gnomad_methods/pull/559/commits/514f634bd4cf31dcdb80a302ac7ba2caeeefffb0

I would lean towards closing this PR and using #559 instead, since it is a superset of these changes